### ISSUE TITLE
WDYN: error creating dummy component 

### ISSUE THREAD

**Issue by AndreaBorgia-Abo** [abapGit/abapGit#4297]
I'm trying to clean up an improper installation of abap2xlsx in a customer system and I get this error from abapGit:

> error creating dummy component 
> Import of object ZDEMO_EXCEL_WDA01 failed 

By "improper" I mean that in this system a CR from another customer was imported into DEV and just left there. 

I've already had to change the trasport layer for the whole package and now I'm trying to reset local objects to a known state, with "advanced, reset local (force pull)". I don't have inactive objects but a couple of classes have some minor (?) differences:
ZCL_EXCEL_WORKSHEET 
ZCL_EXCEL_COMMON

I'd say they are minor because they are not code, rather additional translations which are only present locally and quite possibly not relevant. Also a couple of demos are affected but that doesn't bother me. Since the library appears to be working, I'll move on to extending the customer report to use it but I'd like to clean this up before releasing the library into testing.

Anyhow, how do I fix this? 
I've tried manually recreating the object from SE80 usingi another system as template but it complains  that "the ABAP interface does not exist".

P.S.: I assume this is an abapGit issue since I've had no trouble on three other DEV systems, but please feel free to reassign to abap2xlsx if needed.

**Comment by larshp** [comment]
first, try manually deleting the Web Dynpro Component, and then pull again

notes: "error creating dummy component", looks like it is an error from abapGit, https://github.com/abapGit/abapGit/search?q=error+creating+dummy+component

**Comment by AndreaBorgia-Abo** [comment]
Deleted the webdynpro right after editing the comment above, then pulled, no luck. In the meanwhile I discovered that it's [not a new issue](https://github.com/sapmentors/abap2xlsx/issues/578).
Off to debug we go...

**Comment by AndreaBorgia-Abo** [comment]
I guess I need to "massage" TADIR...
![2020-12-07 12_13_00-Clipboard](https://user-images.githubusercontent.com/25269970/101344577-fa88b080-3885-11eb-87b7-2f1783222f47.png)

**Comment by mbtools** [comment]
Make sure, "only local objects" is turned **off** in the repo settings. Otherwise, abapGit will not recognize objects imported from other systems

![image](https://user-images.githubusercontent.com/59966492/101357732-d5854500-3867-11eb-9323-0c435b203ab8.png)

**Comment by AndreaBorgia-Abo** [comment]
@mbtools it was already off, plus it's giving me hell only on that web dynpro.

**Comment by AndreaBorgia-Abo** [comment]
An update: the interface ZIWCI_DEMO_EXCEL_WDA01 is both existing when you try to create it and nonexisting when you try to find it and delete it. Even force-delete with OO_INTERFACE_DELETE does not help. Whoever originally imported that CR from the other system did me no big favour!

**Comment by AndreaBorgia-Abo** [comment]
So, ended up deleting, releasing, deleting until I got to a "blank" state. Created the package from scratch, pulled again... and the old faithful is still there:
"error creating dummy component 
Import of object ZDEMO_EXCEL_WDA01 failed "

At the point, the main classes have been updated, so I'd call it a day and move the library to the testing system.

**Comment by mbtools** [comment]
My suggestion from https://github.com/abapGit/abapGit/issues/3865#issuecomment-741457446 will help identify if objects came from other systems. 

There's a bug in `ZCL_ABAPGIT_OBJECT_WDYN` (and WDYA). When checking if the WebDynpro exists, it only checks for active version although it should check for any version. I don't think it's related to this but I will also make the error messages more detailed when fixing it (now the error above could be any of 8 exceptions). 

Unless you can reproduce this somehow to test possible solutions, we should close this.

**Comment by AndreaBorgia-Abo** [comment]
As I wrote in the opening post, I know for a fact that all of the package came from another system.
![immagine](https://user-images.githubusercontent.com/25269970/101886132-5bbcc680-3b9b-11eb-9141-74cecfea2fdd.png)

I have re-checked in TADIR, the original sytem was changed. Also, even trying to perform the operation with SE80 gives an error so I'm not sure how much can we fault abapGit for this issue.
